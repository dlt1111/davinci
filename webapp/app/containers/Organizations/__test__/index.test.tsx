import { cleanup } from '@testing-library/react'
import { renderWithReduxAndRouter } from 'app/test/utils'
import Organization from '..'
import injectReducer from 'app/utils/injectReducer'
import reducerOrganization from '../reducer'
import { compose } from 'redux'

afterEach(cleanup)

describe('Organization component test', () => {
  it('should take a snapshot', () => {
    const withOrganizationReducer = injectReducer({
      key: 'organization',
      reducer: reducerOrganization
    })
    const Comp = compose(withOrganizationReducer)(Organization)
    const { asFragment } = renderWithReduxAndRouter(Comp)
    expect(asFragment()).toMatchSnapshot()
  })
})
