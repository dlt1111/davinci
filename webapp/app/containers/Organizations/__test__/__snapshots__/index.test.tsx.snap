// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Organization component test should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="box"
  >
    <div
      class="header"
    >
      <h3
        class="title"
      >
        <div
          class="ant-row"
        >
          <div
            class="ant-col ant-col-20"
          >
            <div
              class="ant-breadcrumb"
            >
              <span>
                <span
                  class="ant-breadcrumb-link"
                >
                  <a
                    href="/account/organizations"
                  >
                    <i
                      aria-label="icon: bars"
                      class="anticon anticon-bars"
                    >
                      <svg
                        aria-hidden="true"
                        class=""
                        data-icon="bars"
                        fill="currentColor"
                        focusable="false"
                        height="1em"
                        viewBox="0 0 1024 1024"
                        width="1em"
                      >
                        <path
                          d="M912 192H328c-4.4 0-8 3.6-8 8v56c0 4.4 3.6 8 8 8h584c4.4 0 8-3.6 8-8v-56c0-4.4-3.6-8-8-8zm0 284H328c-4.4 0-8 3.6-8 8v56c0 4.4 3.6 8 8 8h584c4.4 0 8-3.6 8-8v-56c0-4.4-3.6-8-8-8zm0 284H328c-4.4 0-8 3.6-8 8v56c0 4.4 3.6 8 8 8h584c4.4 0 8-3.6 8-8v-56c0-4.4-3.6-8-8-8zM104 228a56 56 0 1 0 112 0 56 56 0 1 0-112 0zm0 284a56 56 0 1 0 112 0 56 56 0 1 0-112 0zm0 284a56 56 0 1 0 112 0 56 56 0 1 0-112 0z"
                        />
                      </svg>
                    </i>
                    我的组织
                  </a>
                </span>
                <span
                  class="ant-breadcrumb-separator"
                >
                  /
                </span>
              </span>
            </div>
          </div>
          <div
            class="ant-col ant-col-1 ant-col-offset-3"
          >
            <i
              aria-label="icon: plus-circle-o"
              class="anticon anticon-plus-circle-o create"
              tabindex="-1"
            >
              <svg
                aria-hidden="true"
                class=""
                data-icon="plus-circle"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M696 480H544V328c0-4.4-3.6-8-8-8h-48c-4.4 0-8 3.6-8 8v152H328c-4.4 0-8 3.6-8 8v48c0 4.4 3.6 8 8 8h152v152c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8V544h152c4.4 0 8-3.6 8-8v-48c0-4.4-3.6-8-8-8z"
                />
                <path
                  d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
                />
              </svg>
            </i>
          </div>
        </div>
      </h3>
    </div>
  </div>
</DocumentFragment>
`;
